server/sonar-server/src/main/java/org/sonar/server/computation/step/PersistMeasuresStep.java

```diff
@@ -28,6 +28,7 @@
 import java.util.List;
 import java.util.Map;
 import javax.annotation.Nonnull;
+import org.elasticsearch.common.logging.Loggers;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.elasticsearch.common.logging.Loggers'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)